### PR TITLE
Make the datepicker use the native `<input>` element

### PR DESCRIPTION
**Pull request by filiptronicek**

### CODE REVIEW

**Comment by filiptronicek** [comment]
#### 📝 Changed routes:


| [**/create**](https://countdowner-hvc6j89so-filiptronicek.vercel.app/create) |
|:-:|
|<a href="https://countdowner-hvc6j89so-filiptronicek.vercel.app/create">Screenshot of /create</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/rb8tnc0Bwdal9lGBLY2AP6dxsn0QTwNAhmlxxcGOcyf5K0nRd7mGyvsi9BdMRfA8dqGprcIEyQDkWOD57f7G5A.png">(view full size)</a> |







Commit 5c143d4867aa0d3834b712831d824506a2b0960c (https://countdowner-hvc6j89so-filiptronicek.vercel.app).

**Comment by bigint** [comment]
Code LGTM 🚀 

But in the UI I see some text cutoff in date field 🤔 is that the intended one?

Screenshot 2021-10-03 at 9 10 59 PM

File: .eslintrc [hunk 1]
```diff
@@ -12,7 +12,7 @@
     "react/prop-types": "off",
     "react/no-unescaped-entities": "off",
     "no-unused-vars": "warn",
-    "unused-imports/no-unused-imports": "error",
+    "unused-imports/no-unused-imports": "warn",
```
Comment (bigint) [inline .eslintrc:15]: 🤔

Comment (filiptronicek) [reply .eslintrc:15]: Should I leave it at error? I don't want to be too agressive.

**Comment by filiptronicek** [comment]
> Code LGTM 🚀
> 
> But in the UI I see some text cutoff in date field 🤔 is that the intended one?
> 
> Screenshot 2021-10-03 at 9 10 59 PM

This is not intended, didn't find a way to format the inner text somehow else...

File: .eslintrc (hunk 1 above)
Comment (bigint) [reply .eslintrc:15]: IMO it is aggressive, but it helps us to make the app size perfect without any extra imports 🙇🏻

File: components/EventName.tsx [hunk 2]
```diff
@@ -56,15 +59,12 @@ const EventName = (props: {
...
-        <DatePicker
-          className="bg-white dark:bg-[#262A2B] text-black dark:text-white border-2 border-gray-200 dark:border-gray-600 rounded-xl p-2"
-          dateFormat="dd/MM/yyyy"
-          selected={date}
-          showTimeSelect
-          timeIntervals={15}
-          minDate={new Date()}
-          onChange={(val: Date) => {
-            setDate(val);
+        <input
```
[the remark this answers is not in the document]
Comment (filiptronicek) [reply components/EventName.tsx:62]: It has a label lol, idk what's wrong

**Pull request merged**